src/main/java/com/iopipe/IOpipeConfiguration.java

```diff
@@ -361,28 +404,37 @@ public static final IOpipeConfiguration byDefault()
 						Boolean.valueOf(v));
 			}
 			
+			// Just use the standard service connction
+			rv.setRemoteConnectionFactory(new ServiceConnectionFactory());
+			
 			// Determine the URI which is used to collect resources, use the
 			// same region as the AWS service if it is supported.
 			String awsregion = Objects.toString(System.getenv("AWS_REGION"),
-				IOpipeConstants.DEFAULT_REGION);
+				IOpipeConstants.DEFAULT_REGION),
+				origawsregion = awsregion;
 			if (!IOpipeConstants.SUPPORTED_REGIONS.contains(awsregion))
 				awsregion = IOpipeConstants.DEFAULT_REGION;
-		
-			// Build hostname from region
+			
+			// Setup service URL
 			String hostname = (awsregion.equals(
 				IOpipeConstants.DEFAULT_REGION) ?
 				"metrics-api.iopipe.com" :
-				String.format("metrics-api.%s.iopipe.com", awsregion));
-			HttpUrl url;
-			rv.setRemoteConnectionFactory(new ServiceConnectionFactory(
-				(url = new HttpUrl.Builder().
-					scheme("https").
-					host(hostname).
-					addPathSegment("v0").
-					addPathSegment("event").
-					build())));
+				String.format("metrics-api.%s.iopipe.com", awsregion)),
+				surl;
+			rv.setServiceUrl((surl = String.format("https://%s/v0/event",
+				hostname)));
+			
+			_LOGGER.debug(() -> "Remote URL: " + surl);
+			
+			// Profiled events go to any supported region but if the region
+			// is not supported then use a default provided one
+			String purl;
+			rv.setProfilerUrl((purl = String.format(
+				"https://signer.%s.iopipe.com/?extension=.zip",
+				(!IOpipeConstants.SUPPORTED_REGIONS.contains(origawsregion) ?
+				IOpipeConstants.PROFILER_DEFAULT_REGION : origawsregion))));
 			
-			_LOGGER.debug(() -> "Remote URL: " + url);
+			_LOGGER.debug(() -> "Profiler URL: " + purl);
```

Comment:
This should only log if profiling is activated/turned on

Reply:
I can move this message and the default URL to use and put them in `IOpipeConstants`.